ethereum-financial-v2/src/services/pools-balances/pool-balance.srv.ts

```diff
@@ -0,0 +1,389 @@
+import { Logger } from 'winston'
+import { BlockDto } from '../../entity/events'
+import { elapsedTime } from '../../utils/time'
+import BigNumber from 'bignumber.js'
+import * as E from 'fp-ts/Either'
+import { networkAlert, NetworkError } from '../../utils/errors'
+import { PoolBalanceCache } from './pool-balance.cache'
+import { Finding, FindingSeverity, FindingType } from 'forta-agent'
+import { ETH_DECIMALS } from '../../utils/constants'
+
+//  MIN_5 Math.ceil((5 * 60) / 13) = 23
+//  5 minutes in eth blocks
+const MIN_5 = 23
+const PERCENT_10 = 10
+const PERCENT_5 = 5
+
+const PEG_STEP = 0.005
+const PEG_STEP_0_995 = 0.995
+const PEG_STEP_0_98 = 0.98
+
+const WEEK_1 = 60 * 60 * 24 * 7 // 1 week
+const HOURS_24 = 60 * 60 * 24 // 24 hours
```

Comment:
```suggestion
const MINUTES_10 = 60 * 10 // 10 minutes
```

let's use short repeat timeframe for critical alerts